### PR TITLE
Finish assigment-8

### PR DESCRIPTION
**Pull request by kulievtech**

### CODE REVIEW

File: assignments/assignment-8/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>RSVP</title>
+</head>
+<body>
+    <h1>RSVP Form</h1>
+    <h3>Join us for TuranTech's anniversary on April 1, 2024. Please reply to this RSVP by May 1st, 2023. Thank you, we hope to see you soon!</h3>
+
+    <form>
+        <label for="nameInput">Full Name</label>
+        <input type="text" name="fullName" id="nameInput">
```
Comment (aliramazon) [inline assignments/assignment-8/index.html:15]: If asked fullName, the id attribute should be according to it like fullNameInput

File: assignments/assignment-8/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        <label for="emailInput">Email</label>
+        <input type="email" name="email" id="emailInput">
+
+        <h4>Are you attending?</h4>
```
Comment (aliramazon) [inline assignments/assignment-8/index.html:20]: Good heading selection. Liked it

File: assignments/assignment-8/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        <label for="yes">Yes</label>
+        <input type="radio" name="attendingornot" id="yes">
```
Comment (aliramazon) [inline assignments/assignment-8/index.html:23]: id selection is bad. attendanceRadioInput

Comment (aliramazon) [inline assignments/assignment-8/index.html:23]: Bad casing., Should be camelCased, attendingOrNot

Comment (aliramazon) [inline assignments/assignment-8/index.html:23]: Add value as well.

File: assignments/assignment-8/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        <label for="no">No</label>
+        <input type="radio" name="attendingornot" id="no">
```
Comment (aliramazon) [inline assignments/assignment-8/index.html:25]: the same, camelCase name

File: assignments/assignment-8/index.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        <label for="maybe">Maybe</label>
+        <input type="radio" name="attendingornot" id="maybe">
+        <h4>How many family members are coming?</h4>
+        <label for="guestsnumberInput"></label>
```
Comment (aliramazon) [inline assignments/assignment-8/index.html:29]: id should be like numbrOfGuestsInput

File: assignments/assignment-8/index.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        <input type="number" name="guestnumber" id="guestsnumberInput" min="1" max="7" step="2">
+        <br>
+        <label for="mealInput">Choose your favorite meal</label>
```
Comment (aliramazon) [inline assignments/assignment-8/index.html:32]: id should be favoriteMealSelect

**Review by aliramazon — CHANGES_REQUESTED** [review]

File: assignments/assignment-8/index.html (hunk 2 above)
Comment (aliramazon) [reply assignments/assignment-8/index.html]: Kudos for googling and finding out Legends

**Comment by kulievtech** [comment]
Did you review it once again?

File: assignments/assignment-8/assignment-6.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>RSVP</title>
```
Comment (aliramazon) [inline assignments/assignment-8/assignment-6.html:7]: Change to RSVP | Assignment - *,

File: assignments/assignment-8/assignment-6.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+                min="1"
+                max="7"
+                step="2"
+                required
+            />
+            <br />
+            <label for="favoriteMealSelect">Choose your favorite meal</label>
+            <select name="favoritemeal" id="favoriteMealSelect" required>
+                <option value="vegetarian">Vegetarian</option>
+                <option value="nonVegetarian">Non-vegetarian</option>
```
Comment (aliramazon) [inline assignments/assignment-8/assignment-6.html:61]: I said capitalize, not cameCase it.

**Review by aliramazon — APPROVED** [review]

**Pull request merged**